ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -59,7 +61,10 @@ private String parentCommandOptionsOnly() {
         + "--http-listen-port=5001 "
         + "--http-listen-host=localhost "
         + "--chain-id=6 "
-        + "--logging=INFO ";
+        + "--logging=INFO "
+        + "--tls-keystore-file=./keystore.cert "
```

Comment:
usually the extension for pkcs#12 format is either `.pfx` or `.p12`.

Reply:
.cert or .crt is usually for PEM formatted files.

Reply:
done